woocommerce/woocommerce-android
# Disable R8 fullMode

### Description


In [Android Gradle Plugin `8.0`](https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes), the default value for `android.enableR8.fullMode` has been changed from `false` to `true`. The effects of this change is hard to verify because it'd be in runtime.

Considering AGP `8.0` introduced _many_ changes, we (with @jkmassel) decided that it'd be better to revert this flag to `false` and wait for a few releases before enabling it back. This should give us some time to evaluate other changes with AGP and how they impact the releases and also be better prepared before we switch this flag back on.

### Testing instructions

* Run `bundle exec fastlane run configure_apply`
* Verify that `gradle.properties` contains `android.enableR8.fullMode=false`
